kube-controllers/Dockerfile.amd64

```diff
@@ -47,12 +47,13 @@ COPY --from=ubi /lib64/libc.so.6 /lib64/libc.so.6
 COPY --from=ubi /lib64/ld-linux-x86-64.so.2 /lib64/ld-linux-x86-64.so.2
 
 # The following dependencies do not show up when performing ldd on the binary, but are still necessary.
-COPY --from=ubi /lib64/libresolv.so.2 /lib64/libresolv.so.2
```

Comment:
Are you sure we don't need this file?

Reply:
It is unlikely for `libc.so` to require `libresolv.so` to do domain resolution as `libresolv.so` links directly to `libc.so`. If this is the case, system loader has to do cycling loading libc/libresolv shared libraries. I believe `libnss_dns.so` is the plugin who does the generic resolving domain to address work.